Source/DafnyCore/Generic/Reporting.cs

```diff
@@ -14,7 +14,7 @@ public enum ErrorLevel {
   }
 
   public enum MessageSource {
-    Parser, Cloner, RefinementTransformer, Rewriter, Resolver, Translator, Verifier, Compiler, Documentation
+    Parser, Cloner, RefinementTransformer, Rewriter, Resolver, Translator, Verifier, Compiler, Documentation, TestGenerationAuditor
```

Comment:
I'm not sure that the `Auditor` term adds much here. I'd just call this `TestGeneration`. And the checks performed here are just the first phase of test generation.

Reply:
This makes sense. Have made the change